diff -ura goobox/data/org.gnome.Goobox.desktop.in.in goobox_new/data/org.gnome.Goobox.desktop.in.in
--- goobox/data/org.gnome.Goobox.desktop.in.in	2025-03-17 22:06:57.605413980 +0100
+++ goobox_new/data/org.gnome.Goobox.desktop.in.in	2025-03-17 22:10:50.819778893 +0100
@@ -3,15 +3,11 @@
 Comment=Play and extract CDs
 Categories=GNOME;GTK;AudioVideo;Audio;
 Encoding=UTF-8
-Exec=goobox
+Exec=env GTK_THEME=Adwaita:dark goobox
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=goobox
 MimeType=x-content/audio-cdda;
 StartupNotify=true
 Terminal=false
 Type=Application
-X-GNOME-Bugzilla-Bugzilla=GNOME
-X-GNOME-Bugzilla-Product=goobox
-X-GNOME-Bugzilla-Component=general
-X-GNOME-Bugzilla-Version=@VERSION@
 X-GNOME-UsesNotifications=true
diff -ura goobox/help/C/preferences.page goobox_new/help/C/preferences.page
--- goobox/help/C/preferences.page	2025-03-17 22:06:57.605413980 +0100
+++ goobox_new/help/C/preferences.page	2025-03-17 22:09:02.404427080 +0100
@@ -30,7 +30,7 @@
     <gui style="button">Apply</gui> button.
   </p>
 
-  <section>
+  <section id="general">
     <title>General Tab</title>
     <p>This tab allows you to set the following options:</p>
     <list>
@@ -42,7 +42,7 @@
   </section>
 
 
-  <section>
+  <section id="extraction">
     <title>Extraction Tab</title>
     <p>This tab allows you to set the extraction options:</p>
     <list>
diff -ura goobox/help/C/shortcuts.page goobox_new/help/C/shortcuts.page
--- goobox/help/C/shortcuts.page	2025-03-17 22:06:57.605413980 +0100
+++ goobox_new/help/C/shortcuts.page	2025-03-17 22:09:02.405863184 +0100
@@ -1,6 +1,6 @@
 <page xmlns="http://projectmallard.org/1.0/"
       type="topic"
-      id="goobox-keyboard-shortcuts">
+      id="shortcuts">
   <info>
     <link type="guide" xref="index" />
     <title type="sort">1</title>
diff -ura goobox/meson_options.txt goobox_new/meson_options.txt
--- goobox/meson_options.txt	2025-03-17 22:06:57.568413975 +0100
+++ goobox_new/meson_options.txt	2025-03-17 22:09:02.406562987 +0100
@@ -12,6 +12,6 @@
 
 option('disable-libnotify',
   type : 'boolean',
-  value : false,
+  value : true,
   description : 'Disable current track notification'
 )
diff -ura goobox/src/goo-application.c goobox_new/src/goo-application.c
--- goobox/src/goo-application.c	2025-03-17 22:06:57.580413977 +0100
+++ goobox_new/src/goo-application.c	2025-03-17 22:09:02.451427085 +0100
@@ -387,6 +387,7 @@
 	G_APPLICATION_CLASS (goo_application_parent_class)->startup (application);
 
 	g_set_application_name (_("CD Player"));
+	g_set_prgname ("org.gnome.Goobox");
 	gtk_window_set_default_icon_name ("goobox");
 
 	initialize_app_menu (application);
diff -ura goobox/src/goo-player.c goobox_new/src/goo-player.c
--- goobox/src/goo-player.c	2025-03-17 22:06:57.581413977 +0100
+++ goobox_new/src/goo-player.c	2025-03-17 22:09:02.458427085 +0100
@@ -38,7 +38,7 @@
 #define REFRESH_RATE 5
 #define PROGRESS_DELAY 400
 #define QUEUE_SIZE 16384U /*131072U*/
-#define PIPELINE_VOLUME(x) ((x) / 100.0)
+#define PIPELINE_VOLUME(x) ((pow(1.05, x) - 1) / (pow(1.05, 100) - 1))
 
 
 struct _GooPlayerPrivate {
diff -ura goobox/src/goo-window.c goobox_new/src/goo-window.c
--- goobox/src/goo-window.c	2025-03-17 22:06:57.582413977 +0100
+++ goobox_new/src/goo-window.c	2025-03-17 22:09:02.467427086 +0100
@@ -1183,11 +1183,14 @@
 		g_string_append_printf (info, "%s", window->priv->album->title);
 	}
 
-	if (goo_player_get_state (goo_window_get_player (window)) == GOO_PLAYER_STATE_PLAYING)
-		system_notify (window,
-			       "new-track",
-			       window->priv->current_track->title,
-			       (strlen (info->str) > 0) ? info->str : NULL);
+//#ifdef ENABLE_NOTIFICATIONS
+//	if (goo_player_get_state (goo_window_get_player (window)) == GOO_PLAYER_STATE_PLAYING)
+//		system_notify (window,
+//			       "new-track",
+//			       window->priv->current_track->title,
+//			       (strlen (info->str) > 0) ? info->str : NULL);
+//#endif
+
 
 	g_string_free (info, TRUE);
 
@@ -2347,9 +2350,9 @@
 		gtk_scale_button_set_adjustment (GTK_SCALE_BUTTON (button),
 						 gtk_adjustment_new (0.0,
 								     0.0,
-								     100.0,
+								     120.0,
 								     1.0,
-								     10.0,
+								     5.0,
 								     0.0));
 		gtk_scale_button_set_value (GTK_SCALE_BUTTON (button), goo_player_get_audio_volume (window->priv->player));
 		gtk_widget_show (button);
diff -ura goobox/src/preferences.h goobox_new/src/preferences.h
--- goobox/src/preferences.h	2025-03-17 22:06:57.583413977 +0100
+++ goobox_new/src/preferences.h	2025-03-17 22:09:02.495427088 +0100
@@ -54,4 +54,6 @@
 #define PREF_ENCODER_FLAC_COMPRESSION "flac-compression"
 #define PREF_ENCODER_MP3_QUALITY      "mp3-quality"
 
+#define ENABLE_NOTIFICATIONS
+
 #endif /* PREFERENCES_H */
